### PR TITLE
New Reservation Page

### PR DESCRIPTION
**Pull request by ecwu**

### CODE REVIEW

**Review by billzhonggz — COMMENTED** [review]
I think we need to find a method to show the changes on web pages in a visible way.

File: about/reservation.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+<!-- Wrap the rest of the page in another container to center all the content. -->
+
+<div class="container marketing projects">
+
+    <div>
+        <h1>Basic Process</h1>
+        <hr>
+        <div class="alert alert-primary" role="alert">
+            <h4 class="alert-heading"><i class="fa fa-lightbulb-o" aria-hidden="true"></i> Step 1 · Make a decision</h4>
+            <p>We do not have a fixed time for recruiting new members (But usually during the students in school), if you are interested in UIC HCC Computer Society, you can contact us immediately.</p>
```
Comment (jyhi) [inline about/reservation.html]: We do not have a fixed time for recruiting new members (But usually during **the time when** the students in school) ~~,~~ **.** ~~if~~ **If** you are interested in **joining** UIC HCC Computer Society, you can contact us immediately.

File: about/reservation.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            <hr>
+            <p class="mb-0">
+                <a class="btn btn-primary" href="/" role="button">Know More about UIC HCC Computer Society</a>
+            </p>
+        </div>
+        <div class="alert alert-warning" role="alert">
+            <h4 class="alert-heading"><i class="fa fa-calendar" aria-hidden="true"></i> Step 2 · Make a reservation</h4>
+            <p>If you want to join us, the first thing you should do is confirm the interview time with us by email.</p>
```
Comment (jyhi) [inline about/reservation.html]: ~~If you want to join us, t~~ **T**he first thing you ~~should~~ **need to** do is **to** confirm the ~~interview~~ time **of interview** with us by email.

File: about/reservation.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            <hr>
+            <p class="mb-0"><a href="mailto:hi@uichcc.com" class="btn btn-warning">
```
Comment (jyhi) [inline about/reservation.html]: This may lead to spam.

File: TODO.md [hunk 4]
```diff
@@ -19,5 +20,5 @@ There are still some pages that have not been created and are temporarily connec
 - [ ] LCbot UICHCC Edition, `View Details` Link
 
 ## About
-- [ ] Introduction
+- [] Introduction
```
Comment (jyhi) [inline TODO.md]: ?

**Review by jyhi — CHANGES_REQUESTED** [review]

**Review by jyhi — APPROVED** [review]

**Pull request merged**